PowerEditor/src/ScitillaComponent/Buffer.cpp

```diff
@@ -601,31 +601,13 @@ BufferID FileManager::loadFile(const TCHAR * filename, Document doc, int encodin
 		Buffer* buf = _buffers.at(_nbBufs - 1);
 
 		// restore the encoding (ANSI based) while opening the existing file
-		NppParameters *pNppParamInst = NppParameters::getInstance();
-		const NewDocDefaultSettings & ndds = (pNppParamInst->getNppGUI()).getNewDocDefaultSettings();
-		buf->setUnicodeMode(ndds._unicodeMode);

```

Comment:
Pre-emptive note on removal of this call - notice `setUnicodeMode` is (and was even before my changes) called down below in both branches of `if (encoding == -1)` - thus this call is useless.